src/mod/applications/mod_http_cache/mod_http_cache.c

```diff
@@ -1371,13 +1375,27 @@ SWITCH_STANDARD_API(http_cache_get)
 		}
 	}
 
-	filename = url_cache_get(&gcache, profile, session, url, download, refresh, extension, validate_url_extension, use_mime_ext, NULL, pool);
+	switch_event_create_plain(&event, SWITCH_EVENT_CHANNEL_DATA);
+	filename = url_cache_get(&gcache, profile, session, url, download, refresh, extension, validate_url_extension, use_mime_ext, event, pool);
 	if (filename) {
 		stream->write_function(stream, "%s", filename);
-
 	} else {
-		stream->write_function(stream, "-ERR\n");
-		status = SWITCH_STATUS_FALSE;
+		if (event) {
+			const char * curl_status = switch_event_get_header(event, "http_cache_curl_status");
+			if (!zstr(curl_status) && strcmp(curl_status, "0")) {
+				stream->write_function(stream, "-ERR CURL:%s\n", switch_event_get_header(event, "http_cache_curl_error"));
```

Comment:
Could we add the error log here?

Reply:
No need, if I add a log then it will be redundant. The http_get function should also printed the error